clio-server/src/main/scala/org/broadinstitute/clio/server/webservice/SwaggerDirectives.scala

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.clio.server.webservice
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+
+trait SwaggerDirectives {
+
+  lazy val swaggerRoutes: Route = concat(swaggerUiService, swaggerApiDocs)
```

Comment:
It seems that all our routes are lazy so I figured someone knew something I didn't. Doesn't seem obviously necessary since I doubt concat take that long to execute.